src/components/Statistics.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <section class="statistics">
+    <div
+      v-for="item in statistics"
+      :key="item.num"
```

Comment:
Is 'num' unique?

Reply:
In this case is unique, but when enter same num in object therefore will be not unique